src/package.json

```diff
@@ -35,6 +35,7 @@
     "qrcode": "^1.5.4",
     "tailwindcss": "^3.4.10",
     "timeago.js": "^4.0.2",
+    "uuid": "^10.0.0",
```

Comment:
cant find a reference to a import for this package. Should be replaced by `crypto.randomUUID`